src/write_BCs/README.md

```diff
@@ -0,0 +1,11 @@
+## Directions to generate the boundary conditions.
+
+1. Change settings in `config_write_BCs.yml`.
+2. Run run_BCs.sh (`sbatch -p seas_compute -t 4-00:00 --mem 4000 --wrap "bash run_BCs.sh"`).
```

Comment:
Just curious is there a reason you don't just specify the resources in the header of run_BCs.sh?

Reply:
No reason, I just get caught up in my --wrap ways.